src/amp-story-player/amp-story-player-impl.js

```diff
@@ -648,7 +648,20 @@ export class AmpStoryPlayer {
    * @private
    */
   next_() {
-    if (this.currentIdx_ + 1 >= this.stories_.length) {
+    if (
+      this.element_.getAttribute('enable-circular-wrapping') != 'true' &&
+      this.isIndexOutofBounds_(1)
```

Comment:
Hmm what I currently read from this is: 'check if index 1 is out of bounds' , I think we have to either:

(1) change the method name to something like: `isIndexDeltaOutOfBounds(1)` or 
(2) change the method logic so when you call it it reads like this: `isIndexOutOfBounds_(this.currentIndex_ + 1)`

I think I prefer option (2)